tests/org.eclipse.fordiac.ide.test.ui/src/org/eclipse/fordiac/ide/test/ui/PropertySheetHelper.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2009-2019 THALES GLOBAL SERVICES
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *      Obeo - Initial API and implementation
+ *      Prashantkumar Khatri -  Integrated selectPropertyTabItem and
+ *      						createEvent from
+ *      						"org.eclipse.sirius.tests.swtbot.support.api.editor.SWTBotSiriusHelper"
+ */
+package org.eclipse.fordiac.ide.test.ui;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Event;
+import org.eclipse.swt.widgets.Widget;
+import org.eclipse.swtbot.eclipse.finder.SWTWorkbenchBot;
+import org.eclipse.swtbot.eclipse.finder.matchers.WidgetMatcherFactory;
+import org.eclipse.swtbot.swt.finder.SWTBot;
+import org.eclipse.swtbot.swt.finder.finders.UIThreadRunnable;
+import org.eclipse.swtbot.swt.finder.results.BoolResult;
+import org.eclipse.swtbot.swt.finder.utils.SWTUtils;
+import org.eclipse.ui.internal.views.properties.tabbed.view.TabbedPropertyList;
+import org.hamcrest.Matcher;
+import org.hamcrest.Matchers;
+
+public class PropertySheetHelper {
+
+	/**
+	 * SWTWorkbenchBot
+	 */
+	private static SWTWorkbenchBot bot = new SWTWorkbenchBot();
+
+	/**
+	 * Select the tab with the name label in the property views.
+	 *
+	 * @param label         Label to find.
+	 * @param propertiesBot the bot corresponding to the property view.
+	 * @return true if the property tab is found, false otherwise
+	 */
+	@SuppressWarnings({ "unchecked" })
```

Comment:
what does this suppress warning do?

Reply:
We have utilized methods from the Eclipse Sirius repository and have endeavored to maintain their integrity by not making any modifications to them, so I have not changed anything in the method. I have checked to remove the suppress warning, it does nothing, one warning we are receiving is that is related to we are using "org.eclipse.ui.views.properties.tabbed_3.10.300.v20240424-0956.jar" here, so need to a restriction warning here.